airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -343,7 +373,7 @@ public void create(final Map<String, String> allLabels,
         .getStatus();
 
     final var containerState = podStatus
-        .getContainerStatuses()
+        .getInitContainerStatuses()
```

Comment:
Shouldn't we still check the `containerStatus`?

Reply:
The main-container doesn't start until after the init-container exited with a non-error status.  

Waiting on the non init-container would result in [this throwing an exception](https://github.com/airbytehq/airbyte/blob/9ab7f46c5dcb0f8738898a74ceaf483ebc8f0756/airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java#L380-L382) and exiting as the main container cannot exist until after the init-container has completed the _file exists_ check successfully. 

The wait the code currently works, it checks the status of the container _before_ attempting to copy the files over.